crengine/src/lvtinydom.cpp

```diff
@@ -14112,40 +14141,37 @@ ldomDocumentWriterFilter::ldomDocumentWriterFilter(ldomDocument * document, bool
 , _curFosteredNode(NULL)
 , _lastP(NULL)
 {
-    if (gDOMVersionRequested >= 20200824) {
+    if (_document->getDOMVersionRequested() < 20200824) {
```

Comment:
Why change the comparison order?  You then have to indent what follows, which cause some diff (and I wonder if you changed something - I guess not - and it will pollute git blame).

Reply:
I don't think about 'git blame'. I think about code readability. Do you propose to revert this?

Reply:
>  I can revert this.

Yes please, thanks for git blame :)